pkg/postgres/walker/walker.go

```diff
@@ -455,33 +455,11 @@ func handleStruct(ctx walkerContext, schema *Schema, original reflect.Type) {
 			// The return values is a slice of interfaces that are nil type pointers
 			if structField.Tag.Get("protobuf_oneof") == "" {
 				panic("non-oneof interface is not handled")
-
-			}
-			ptrToOriginal := reflect.PtrTo(original)
-
-			methodName := fmt.Sprintf("Get%s", field.Name)
-			oneofGetter, ok := ptrToOriginal.MethodByName(methodName)
-			if !ok {
-				panic("didn't find oneof function, did the naming change?")
```

Comment:
what happened with this panic?

Reply:
I think we `panic` here only if the generated code is not valid. We used getter to find field type (interface) -> what can also be fetched by `structField.Type`. 🤷